### PR TITLE
v0.14 roadmap update

### PR DESCRIPTION
**Pull request by prasek**
v0.14 roadmap update through v1.0

Signed-off-by: Phil Prasek <prasek@gmail.com>

I have:
- [x] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable test` to ensure this PR is ready for review.

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -259,70 +258,163 @@ Any dates listed below and the specific issues that will ship in a given milesto
...
+* Composition
+  * APIs to v1beta1
+  * Propagating updates made to a claim to its underlying composite resource
+  * Hardening and robustness fixes
 
-* OAM - Applications
-  * Move out of core into separate package
-  * Updates the runtime to the latest tagged version (includes HealthScope)
-  * Support for Scope in AppConfig controller
+* Package Manager
```
Comment (hasheddan) [inline ROADMAP.md:334]: Might be good to mention Provider controller config here also

Comment (prasek) [reply ROADMAP.md:342]: Good catch, updated!

**Comment by prasek** [comment]
should be good to go now

File: ROADMAP.md [hunk 2]
```diff
@@ -259,107 +265,187 @@ Any dates listed below and the specific issues that will ship in a given milesto
...
+* Open Application Model (OAM)
+  * Moved AppConfig controller out of core
+    * Install via: `helm install crossplane` with the `--set alpha.oam.enabled=true` flag
+  * Enhance health scope with informative health condition [#194](https://github.com/crossplane/oam-kubernetes-runtime#194)
+  * Add component webhook to support workload definition type [#198](https://github.com/crossplane/oam-kubernetes-runtime#198)
+  * Add health check support for containerized.standard.oam.dev in Health [#214](https://github.com/crossplane/oam-kubernetes-runtime#214)
+  * Run with fewer privileges [#228](https://github.com/crossplane/oam-kubernetes-runtime/pull/228)
+  * Hardening and robustness enhancements towards v1beta1 quality
+
+## v0.14 - v1beta1 APIs in core, hardening, robustness (short release)
```
Comment (jbw976) [inline ROADMAP.md:335]: i wonder if next time we should consider reverse sorting this list, so the newest milestones are on top

File: ROADMAP.md [hunk 3]
```diff
@@ -259,107 +265,187 @@ Any dates listed below and the specific issues that will ship in a given milesto
...
+  * Migrate from Jenkins pipeline to GitHub Actions [#260](https://github.com/crossplane/oam-kubernetes-runtime/pull/260)
+  * CRD discovery mechanism [#261](https://github.com/crossplane/oam-kubernetes-runtime/pull/261)
+
+## v1.0 release candidate (Dec 2020)
+
+* Hardening and cleanup for v1.0
+  * Leader election for all controllers [#5](https://github.com/crossplane/crossplane/issues/5)
+  * Prometheus metrics for all binaries [#314](https://github.com/crossplane/crossplane/issues/314)
+  * crossplane-runtime to v1.0
+  * Remove deprecated `KubernetesApplication`, `KubernetesTarget`, `KubernetesCluster`
```
Comment (jbw976) [inline ROADMAP.md:381]: this is in v0.14

File: ROADMAP.md (hunk 2 above)
Comment (prasek) [reply ROADMAP.md:335]: agree, thinking the same.

File: ROADMAP.md (hunk 3 above)
Comment (prasek) [reply ROADMAP.md]: good catch ... fixed!

File: ROADMAP.md (hunk 2 above)
Comment (prasek) [reply ROADMAP.md:335]: or maybe a toc so you can just jump, so you're not looking at the longer term roadmap the very first thing

Comment (prasek) [reply ROADMAP.md:335]: will add a toc, just a second

Comment (prasek) [reply ROADMAP.md]: @jbw976 added toc and reordered, how does that look?

**Review by jbw976 — APPROVED** [review]

**Comment by jbw976** [comment]
@prasek looks like this branch is out-of-date with master now, can you rebase?

**Comment by prasek** [comment]
@jbw976 all set, should be good to go, thx!

**Pull request merged**